benchmarks/imagenet/vitb16/finetune_eval.py

```diff
@@ -35,6 +35,9 @@ def __init__(
         super().__init__(
             model, batch_size_per_device, feature_dim, num_classes, topk, freeze_model
         )
+        # Add path dropout.
+        model.__dict__["backbone"].__dict__["vit"].drop_path_rate = 0.1
```

Comment:
I just checked and it looks like this is more complicated than expected. While this change sets `model.backbone.vit.drop_path_rate` to `0.1` it doesn't actually add path dropout. The reason is that timm creates the path dropout layers when the vision transformer class is created ([here](https://github.com/huggingface/pytorch-image-models/blob/88889de923149533593482879a564e7f7e0dabbd/timm/models/vision_transformer.py#L151) and [here](https://github.com/huggingface/pytorch-image-models/blob/88889de923149533593482879a564e7f7e0dabbd/timm/models/vision_transformer.py#L161)). Changing the parameter afterwards does not affect the layers anymore. An additional problem is that the dropout parameter is scaled with every layer [here](https://github.com/huggingface/pytorch-image-models/blob/88889de923149533593482879a564e7f7e0dabbd/timm/models/vision_transformer.py#L504). 

I am not sure what the best way is to enable path dropout after creating the model. Our setup does not support his usecase well. I guess we would have to add a `set_path_dropout` method to the model and call it from here. To set the path dropout we would have to go through the model and create the layers and make sure the drop rate is correctly scaled. Maybe we can add a todo comment for now?